src/server/routes/oauth.ts

```diff
@@ -148,7 +209,8 @@ const authenticationHandler = async (
 	res: ResponseWithRequestState,
 	tokenSet: TokenSet,
 	authState: AuthorizationState,
-) => {
+	openIdClient: OpenIdClient,
+): Promise<void> => {
```

Comment:
This is not a problem introduced in this PR but as I think this function is too long. 
We should consider if there's opportunities to break off self contained parts of it into separate functions with descriptive names.
Obviously I'm not saying we should do it in this PR as it would delay the merging of this and it would add a lot of noise to the diff

Reply:
@coldlink and I were just talking about this. It's way too long!